src/serial/SerialInputHandler.cpp

```diff
@@ -621,6 +645,20 @@ authtoken [<token>]
 )",
   _handleAuthtokenCommand,
 };
+static const SerialCmdHandler kLcgOverrideCmdHandler = {
+  "lcg_override",
+  R"(lcg_override
+  Get the overridden LCG endpoint.
+
+lcg_override [<domain>]
```

Comment:
I don't see a way to clear the override here. A user or dev messing with this would currently need to reset their whole device if they set one.

Perhaps if you use `lcg_override clear` , or if we add the domain check and it fails, it'll clear it?